packages/core/src/routes/swagger.ts

```diff
@@ -101,16 +112,25 @@ export default function swaggerRoutes<T extends AnonymousRouter, R extends Route
   allRouters: R[]
 ) {
   router.get('/swagger.json', async (ctx, next) => {
+    const routeStatusCodes = load(
+      readFileSync('static/yaml/route-status-codes.yaml', 'utf-8')
+    ) as RouteStatusCodes;
```

Comment:
- don't use as here since it's not safe
- the yaml should be per-route. i.e. one yaml for one route